docs/overview.rst

```diff
@@ -0,0 +1,350 @@
+been implemented yet:
+
+  - it is not possible to write custome types in C. There is already a WIP
+    branch to address this issue
+
+  - only a handful of the original Python/C functions has been ported to
+    HPy. Porting most of them is straighforward, so for now the priority is to
+    work on the "hard" features to prove that the HPy approach works, and we
+    port new functions as needed
+
+  - the debug mode simply does not exist (yet!)
+
+  - there is not a standard/easy way to integrate HPy in a
+    ``distutils/setuptools`` based workflow. The only way to compile HPy
+    extensions right now is to manually close the git repo and tweak your
+    ``setup.py``. Eventually, we plan to offer a workflow which integrates
+    seamlessly with ``pip``, ``setuptools``, etc.
+
+  - there is no integration with Cython. The medium-term plan is that Cython
+    automatically generates HPy-compatible C code
+
+
+Early benchmarks
+-----------------
+
+To validate our approach, we ported a simple yet performance critical module
+to HPy. We choose `ultrajson <https://github.com/pyhandle/ultrajson-hpy>`_
+because it is simple enough to require porting only a handful of API
+functions, but at the same time it is performance critical and it does many
+API calls during the parsing of a JSON file.
+
+This `blog post <https://morepypy.blogspot.com/2019/12/hpy-kick-off-sprint-report.html>`_
+explains the results in more detail, but they can be summarized as follows:
+
+  - ``ujson-hpy`` compiled with the CPython ABI is as fast as the original
+    ``ujson``.
+
+  - A bit surprisingly, ``ujson-hpy`` compiled with the HPy Universal ABI is
+    only 10% slower on CPython.  We need more evidence than a single benchmark
+    of course, but if the overhead of the HPy Universal ABI is only 10% on
+    CPython, for many projects it might be small enough to avoid caring about
+    compiling/distributing extensions with the CPython ABI, in the long term.
+
+  - On PyPy, ``ujson-hpy`` runs 3x faster than the original ``ujson``. Note
+    the HPy implementation on PyPy is not fully optimized yet, so we expect
+    even bigger speedups eventually.
+
+
+Projects involved
+-----------------
+
+HPy was born during EuroPython 2019, were a small group of people started to
+discuss about the problems of the Python/C API and how it would be nice to
+have a way to fix them.  Since then, it gathered the attention and interest of
+people which are involved in many projects of the Python ecosystem.  The
+following is a (probably incomplete) list of projects whose core developers
+are involved in HPy, in one way or the other.  The mere presence in this list
+does not mean that the project as a whole endorse or recognize HPy in any way,
+just that some of the people involved contributed to the
+code/design/discussions of HPy:
+
+  - PyPy
+
+  - CPython
+
+  - Cython
+
+  - Numpy (XXX for mattip: do you think it is good/correct to include numpy here?)
```

Comment:
Probably not. I didn't discuss it on any public mailing list nor with the NumPy steering council.

Reply:
fixed by 4cbf1ee